cmd/osmosisd/cmd/balances_from_state_export.go

```diff
@@ -179,11 +179,6 @@ Example:
 
 			authGenesis := authtypes.GenesisState{}
 			clientCtx.Codec.MustUnmarshalJSON(genState["auth"], &authGenesis)
-			accounts, err := authtypes.UnpackAccounts(authGenesis.Accounts)
```

Comment:
Was this removed on purpose?

Reply:
Yes, this was unused other than unpacking

Reply:
I just double-checked, and I think the two lines above can also be removed:
```
authGenesis := authtypes.GenesisState{}
clientCtx.Codec.MustUnmarshalJSON(genState["auth"], &authGenesis)
```

Generally, I'm having difficulty understanding what's happening in that command. We might want to consider having an issue to determine what is the need for it and potentially refactor